e2e/tools/validator/validator-regression.yaml

```diff
@@ -0,0 +1,30 @@
+log_level: info # Logging level, defaults is warn
+
+remote:
+  host: 192.168.1.1 # IP address or hostname of the VM
+  port: 22 # SSH port, default is 22
+  username: user # SSH username
+  password: yourpassword # SSH password
+  pkey: ~/.ssh/id_rsa # Path to SSH private key
+
+metal:
+  vm:
+    pid: 123456 # Process ID for the KVM process running on metal
+
+prometheus:
+  job:
+    vm: vm  # Job name for virtual machine metrics, default is vm
+    metal: metal  # Job name for metal metrics, default is metal
+
+  url: http://localhost:9090 # Prometheus server URL
+  rate_interval: 60s  # Rate interval for Promql, default is 20s, typically 4 x $scrape_interval
```

Comment:
Explicitly using rate interval as 60s because:
```
Prometheus scrape Interval = 3s
Data points for 12s Interval(i.e 4* scrape interval) = 12/3 = 4 data points
Data points for 60s interval = 60/3 = 20 data points
```
With 20 data points, we get a smoother and more reliable estimate. When comparing two `sum(rate(...))` a stable rate reduces the variability in MAE calculations leading to more accurate assessments.